src/main/java/com/microsoft/aad/msal4j/ServiceExceptionClassification.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+/**
+ * Details about the cause of an {@link MsalInteractionRequiredException}, giving a hint about the
+ * user can expect when they go through interactive authentication
+ */
+public enum ServiceExceptionClassification {
```

Comment:
>ServiceException [](start = 12, length = 16)

is this the right name? Why ServiceException ? Why not InteractionRequiredClassification ?

Reply:
You are right, it should be InteractionRequiredReason, as per Bogdan's latest email.